lib/Bio/EnsEMBL/DataCheck/DbCheck.pm

```diff
@@ -372,66 +372,26 @@ sub get_old_dba {
     my $message = 'Specified database does not exist';
     $dbh = $self->test_db_connection($uri, $params{'-DBNAME'}, $message);
   } elsif ($group =~ /compara|ontology/) {
-    my $dbname = $self->dba->dbc->dbname;
-    my $current = $mca->schema_version;
-    $dbname =~ s/$current/$db_version/; # Ensembl version
-    my ($eg_current, $eg_version) = ($current-53, $db_version-53);
-    $dbname =~ s/$eg_current/$eg_version/; # EG version
-    $params{'-DBNAME'} = $dbname;
-    my $message = 'Previous version of database does not exist';
-    $dbh = $self->test_db_connection($uri, $params{'-DBNAME'}, $message);
+    ($params{'-DBNAME'}, $dbh) = $self->old_dbname_from_version(
```

Comment:
Is it sensible to group the testing from `old_dbname_from_version` to code in `old_dbname_from_metadata`
Because the latter do a test on group as well. Hence reducing complexity of the if then else ?

Reply:
Agreed, change committed.